pkg/proxy/ipvs/proxier.go

```diff
@@ -472,6 +496,7 @@ func NewProxier(ipt utiliptables.Interface,
 		nodePortAddresses:     nodePortAddresses,
 		networkInterfacer:     utilproxy.RealNetwork{},
 		gracefuldeleteManager: NewGracefulTerminationManager(ipvs),
+		detectLocal:           detectLocal,
```

Comment:
Thoughts on `localDetector`? I initially assume `detectLocal` is a bool

Reply:
Ack. Will change to localDetector to minimize confusion.

Reply:
Done.